luzer/io.cc

```diff
@@ -0,0 +1,91 @@
+/*
+ * SPDX-License-Identifier: ISC
+ *
+ */
+#include <vector>
+#include <string>
+#include <cstdint>
+/**
+ * Okay, we all know this is bad, but unless we want to include third-party
+ * headers or libs to do crossplatform IO (damn Windows cannot into readdir)
+ * we better use whatever libfuzzer... shyly gives to us with no guarantees.
+ * Remember - those things do not have ATTRIBUTE_INTERFACE in LF's codebase.
+ * Bu-u-u-ut libfuzzer is pretty much in maintenance mode so I think it's
+ * safe.
+ * What's worse than using non-public-API is using C++. But this project already
+ * uses clang++ with 'fuzzed_data_provider.cc'. Hey, libfuzzer IS written in C++.
```

Comment:
I don't like the approach. Please rewrite to C.

Reply:
Would you be okay with C-binding-to-a-mangled-Cpp-symbol-of-libfuzzer or do you mean "write new, non-libfuzzer IO code in C without using `fuzzer::ReadDirToVectorOfUnits()`? I could do the former easily, but the latter would require significant work to be cross-platform.

Reply:
> Would you be okay with C-binding-to-a-mangled-Cpp-symbol-of-libfuzzer or do you mean "write new, non-libfuzzer IO code in C without using fuzzer::ReadDirToVectorOfUnits()?

I would prefer a plain C variant, but C-binding-to-a-mangled-Cpp-symbol-of-libfuzzer would be okay too.